forge-game/src/main/java/forge/game/card/CardProperty.java

```diff
@@ -607,9 +607,12 @@ public static boolean cardHasProperty(Card card, String property, Player sourceC
             }
         } else if (property.startsWith("CanBeTargetedBy")) {
             final String def = property.substring(15);
-            final SpellAbility targetingSA = AbilityUtils.getDefinedSpellAbilities(source, def, spellAbility).get(0);
-            if (!targetingSA.canTarget(card)) {
-                return false;
+            SpellAbility targetingSA = AbilityUtils.getDefinedSpellAbilities(source, def, spellAbility).get(0);
+            while (targetingSA != null) {
+                if (targetingSA.usesTargeting() && !targetingSA.canTarget(card)) {
+                    return false;
+                }
+                targetingSA = targetingSA.getSubAbility();
             }
```

Comment:
shouldn't that iterate all SpellAbilities first before returning false?

What if:
* Target Player
* and Target Creature that Player controls

?

Reply:
this got implemented for _Feather, Radiant Arbiter_:
> Whenever you cast a noncreature spell that targets only CARDNAME, you may choose any number of other creatures that spell could target and pay {2} for each of those creatures.
so if it would be illegal target even partly it can't be copied imo